packages/atomic/src/components/atomic-relevance-inspector/atomic-relevance-inspector.tsx

```diff
@@ -0,0 +1,49 @@
+import {Component, h, State, Prop} from '@stencil/core';
+import {
+  RelevanceInspector,
+  RelevanceInspectorState,
+  Unsubscribe,
+  buildRelevanceInspector,
+  Engine,
+} from '@coveo/headless';
+
+@Component({
+  tag: 'atomic-relevance-inspector',
+  shadow: true,
+})
+export class AtomicRelevanceInspector {
+  @State() state!: RelevanceInspectorState;
+  @Prop() engine!: Engine;
+
+  private relevanceInspector!: RelevanceInspector;
+  private unsubscribe: Unsubscribe = () => {};
+
+  constructor() {
+    this.relevanceInspector = buildRelevanceInspector(this.engine, {
+      initialState: {
+        // TODO: add enable/disable mechanism
+        enabled: false,
+      },
+    });
+    this.unsubscribe = this.relevanceInspector.subscribe(() =>
+      this.updateState()
+    );
+  }
+
+  public disconnectedCallback() {
+    this.unsubscribe();
+  }
+
+  private updateState() {
+    this.state = this.relevanceInspector.state;
+  }
+
+  public render() {
+    if (!this.state.isEnabled) {
+      return;
+    }
+
+    // TODO: Display data in a cleaner manner
+    return <div>{JSON.stringify(this.relevanceInspector.state)}</div>;
```

Comment:
I might misunderstand the purpose of this PR, but why display the data in the HTML? I thought I would be shown in the console

Reply:
Nvm, it gets logged to the console in the other PR 😄  but still, why is it in the HTML?

Reply:
It will at some point be displayed in a nice table, I can remove it for now